tests/passes/group_to_seq/basic_test.futil

```diff
@@ -0,0 +1,66 @@
+//-p canonicalize -p group2seq
```

Comment:
Split this into two different tests:
1. That tests if splitting works
2. If splitting does not break up chain_plus_adders